src/components/Auth/Title.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {
+  Button, Form, FormGroup, Label, Input, FormText,
+} from 'reactstrap';
+import { Link } from 'react-router-dom';
+
+export default function Title() {
```

Comment:
Where are we using this function in our code?

Reply:
These codes contain the small logo in case someone wants to use it.

Reply:
But, is there someone who wants to use it? 
If not, then you can remove. 
If, yes, you can give more descriptions.